android/src/main/java/io/ably/flutter/plugin/AblyMethodCallHandler.java

```diff
@@ -205,10 +203,7 @@ public void success(@Nullable Object result) {
 
                 @Override
                 public void error(String errorCode, @Nullable String errorMessage, @Nullable Object errorDetails) {
-                  System.out.println(errorDetails);
-                  if (errorMessage != null) {
-                    result.error("40000", String.format("Error from authCallback: %s", errorMessage), errorDetails);
-                  }
+                  result.error("40000", String.format("Error from authCallback: %s", errorMessage), errorDetails);
```

Comment:
I should've removed this change in this PR, because this `result.error` usage is removed completely in a future PR: https://github.com/ably/ably-flutter/pull/231/files because it causes a bug.